web/src/ProviderListPage.js

```diff
@@ -204,6 +204,28 @@ class ProviderListPage extends BaseListPage {
           );
         },
       },
+      {
+        title: i18next.t("general:Region"),
+        dataIndex: "region",
+        key: "region",
+        width: "120px",
+        sorter: (a, b) => a.region !== undefined ? a.region.localeCompare(b.region) : "",

```

Comment:
Like `sorter: (a, b) => a.providerUrl.localeCompare(b.providerUrl),` should be enough